dojo/tools/pmd/parser.py

```diff
@@ -0,0 +1,66 @@
+import io
+import csv
+import hashlib
+from dojo.models import Finding
+
+
+class PmdParser(object):
+
+    def get_scan_types(self):
+        return ["PMD Scan"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return scan_type
+
+    def get_description_for_scan_types(self, scan_type):
+        return "CSV Report"
+
+    def get_findings(self, filename, test):
+        dupes = dict()
+
+        content = filename.read()
+        if type(content) is bytes:
+            content = content.decode('utf-8')
+        reader = csv.DictReader(io.StringIO(content), delimiter=',', quotechar='"')
+        csvarray = []
+
+        for row in reader:
+            csvarray.append(row)
+
+        for row in csvarray:
+            finding = Finding(test=test)
+            finding.title = row["Problem"]
+            if row["Priority"] == "5":
+                priority = "Critical"
+            elif row["Priority"] == "4":
+                priority = "High"
+            elif row["Priority"] == "3":
+                priority = "Medium"
+            elif row["Priority"] == "2":
+                priority = "Low"
+            elif row["Priority"] == "1":
+                priority = "Info"
+            else:
+                priority = row["Priority"]
```

Comment:
priority will be a number like "1" here.
```suggestion
                priority = "Info"
```